pkg/record/schema/proto/schema_test.go

```diff
@@ -214,41 +226,41 @@ func TestStructDescriptor_Fields(t *testing.T) {
 		mainDescriptor: "",
 		getDescriptor: func(t *testing.T, s schema.Schema) StructDescriptor {
 			d, ok := s.Descriptors()[1].(StructDescriptor)
-			assert.True(t, ok, "expected %T, got %T", StructDescriptor{}, s.Descriptors()[1])
-			assert.Equal(t, "AllTypes", d.Name())
+			is.True(ok)
+			is.Equal("AllTypes", d.Name())
 			return d
 		},
 		assertDescriptor: func(t *testing.T, descriptor StructDescriptor) {
 			fields := descriptor.Fields()
 
-			assert.Equal(t, 19, len(fields))
+			is.Equal(19, len(fields))
 			for i, f := range fields {
-				assert.Equal(t, fmt.Sprintf("f%d", i+1), f.Name())
+				is.Equal(fmt.Sprintf("f%d", i+1), f.Name())
 
 				if i < 19 {
-					assert.Equal(t, i+1, f.Index())
+					is.Equal(i+1, f.Index())
 				} else {
-					assert.Equal(t, (i+1)*10, f.Index())
+					is.Equal((i+1)*10, f.Index())
 				}
 
 				switch i {
 				case 15:
 					_, ok := f.Descriptor().(StructDescriptor)
-					assert.True(t, ok, "expected %T, got %T", StructDescriptor{}, f.Descriptor())
+					is.True(ok)
 				case 16:
 					_, ok := f.Descriptor().(ArrayDescriptor)
-					assert.True(t, ok, "expected %T, got %T", ArrayDescriptor{}, f.Descriptor())
+					is.True(ok)
 				case 17:
 					_, ok := f.Descriptor().(MapDescriptor)
-					assert.True(t, ok, "expected %T, got %T", MapDescriptor{}, f.Descriptor())
+					is.True(ok)
 				case 18:
 					_, ok := f.Descriptor().(EnumDescriptor)
-					assert.True(t, ok, "expected %T, got %T", EnumDescriptor{}, f.Descriptor())
+					is.True(ok)
 
 				default:
 					// first 15 fields should be primitive types
 					_, ok := f.Descriptor().(PrimitiveDescriptor)
-					assert.True(t, ok, "expected %T, got %T", PrimitiveDescriptor{}, f.Descriptor())
+					is.True(ok)
```

Comment:
Alternatively, we could also use `is.Equal(reflect.TypeOf(PrimitiveDescriptor{}), reflect.TypeOf(f.Descriptor()))